robottelo/vm.py

```diff
@@ -16,7 +16,7 @@
 from robottelo.common import conf, ssh
 
 
-BASE_IMAGES = ('rhel7', 'rhel65', 'rhel66')
+BASE_IMAGES = ('rhel71', 'rhel70', 'rhel66', 'rhel65')
```

Comment:
#1 - good catch. #2 is comment, #3 is actually changed in this PR.
Regarding #1, what's prefered rhel version? The latest one ```rhel71```, or maybe it's better to leave it as it was (```rhel70```) e.g. for compatibility issues?